### DISCUSSION TITLE
GPUMD Interface

### DISCUSSION THREAD

**Discussion by Youhaojen** [deepmodeling/deepmd-kit#2535] [Ideas]
Dear DeePMD Developers,

I would like to express my appreciation for the excellent performance of DeePMD on Lammps. It has proven to be a valuable tool in our work. However, I was curious if there is a potential opportunity to extend its capabilities by creating an interface for GPUMD.

Best regards,
Hao-Jen You

**Comment by njzjz** [comment]
I guess it is better to develop it in the GPUMD if it does not provide an interface for third-party potentials. cc @brucefan1983

**Comment by brucefan1983** [comment]
> Dear DeePMD Developers,
> 
> I would like to express my appreciation for the excellent performance of DeePMD on Lammps. It has proven to be a valuable tool in our work. However, I was curious if there is a potential opportunity to extend its capabilities by creating an interface for GPUMD.
> 
> Best regards, Hao-Jen You

So I guess you hope to use a trained DP model within GPUMD, possibly because you need some molecular dynamics features in GPUMD? Please correct me if I misunderstand you. 

This would be a difficult task for me as I am not familiar with tensorflow that is an essential building block of DP. I also hope to keep the default version of GPUMD as a standalone package that does not depend on any third-party ones such as tensorflow. So even if this is to be done, it will be created and mainted by some volunteer(s) and will be set as an optional choice for users.

**Reply by wanghan-iapcm** [reply]
The interface of an MD package to deepmd-kit is pure C/C++, no tensorflow involved.
